create_analysis_tags_page.py

```diff
@@ -0,0 +1,61 @@
+"""
+TODO: create database for Run3 planning
+"""
+import sys
+import sqlite3
+import logging
+#pylint: disable=wrong-import-position,import-error
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+from utils import tags
```

Comment:
Please move `from utils import tags` outside `#pylint: ...` comments. These comments are to tell pylint not to check McM import from AFS.